<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialSettings</class>
 <widget class="QDialog" name="SerialSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>257</width>
    <height>197</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serial Configuration</string>
  </property>
  <property name="windowIcon">
   <iconset resource="intercom.qrc">
    <normaloff>:/res/settings.png</normaloff>:/res/settings.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>201</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Bit Settings</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>160</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lbBaud">
       <property name="text">
        <string>Baud Rate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbBaudRate"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>160</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lbParity">
       <property name="text">
        <string>Parity</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbParity"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>160</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="lbStopBits">
       <property name="text">
        <string>Stop Bits</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbStopBits"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>239</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="bnOK">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bnSave">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bnCancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="intercom.qrc"/>
 </resources>
 <connections/>
</ui>
